modules/survey_accounts/php/addsurvey.class.inc

```diff
@@ -61,6 +61,26 @@ class AddSurvey extends \NDB_Form
     {
         $db = \Database::singleton();
 
+        // Check that all required fields are present.
```

Comment:
@johnsaigle Are you sure change to this file and  `modules/survey_accounts/test/survey_accountsTest.php` goes into this PR? Look like it should be in another PR.